variables.tf

```diff
@@ -321,3 +321,20 @@ variable "max_instance_lifetime" {
   default     = 0
 }
 
+variable "instance_refresh_minimum_health_percentage" {
```

Comment:
Please remame this variable into `instance_refresh_min_healthy_percentage`

Reply:
done! @spatel96 can you set as resolved?